androidApp/src/main/java/com/mbta/tid/mbta_app/android/map/HomeMapView.kt

```diff
@@ -258,6 +258,11 @@ fun HomeMapView(
 
             val locationProvider = remember { PassthroughLocationProvider() }
 
+            MapEffect(true) { map ->
+                map.mapboxMap.addOnMapClickListener { point -> handleStopClick(map, point) }
+                map.location.setLocationProvider(locationProvider)
+            }
+
```

Comment:
After completing the onboarding flow, the `LocationDataManager.currentLocation` already has the current location when the `HomeMapView` is being initialized, so if we don't `setLocationProvider` until after we've already `collect`ed, the first location will be completely lost, the puck won't show, and recentering will not work. (This was not fun to track down.)